welcome-message/index.js

```diff
@@ -1,22 +1,103 @@
 const superstore = require('superstore-sync');
-
-import { $ } from '../utils';
+const expander = require('o-expander');
+const oViewport = require('o-viewport');
 
 const STORAGE_KEY = 'n-welcome-message-seen';
+const HAS_MINIMIZED = 'n-welcome-message-collapsed';
 const TEST_KEY = 'n-welcome-message-test';
 const TEST_VAL = 'can-store';
 
+let fixedEl;
+let fixedElHeight;
+let staticEl;
+let viewportHeight;
+
 function hasLocalStorage () {
 	superstore.local.set(TEST_KEY, TEST_VAL);
 	const retrieved = superstore.local.get(TEST_KEY);
 	superstore.local.unset(TEST_KEY);
 	return TEST_VAL === retrieved && superstore.isPersisting();
 }
 
-function init () {
-	const fixedEl = $('.n-welcome-message--fixed');
-	const staticEl = $('.n-welcome-message--static');
+function userHasMinimized () {
+	return Boolean(superstore.local.get(HAS_MINIMIZED));
+}
+
+function setExpander () {
+	const buttonContainer = document.createElement('div');
+	const button = document.createElement('button');
+	const buttonClass = 'n-welcome-banner__button--toggler';
+	const expandableContent = fixedEl.querySelector('.o-expander__content');
+	buttonContainer.setAttribute('class', 'n-welcome-banner__column');
+	button.setAttribute('data-trackable', 'toggled');
+	button.setAttribute('class', buttonClass);
+	buttonContainer.appendChild(button);
+	expandableContent.parentNode.appendChild(buttonContainer);
+
+	if (userHasMinimized()) {
+		expandableContent.setAttribute('aria-hidden', true);
+		expandableContent.setAttribute('className', expandableContent.className.replace('o-expander__content--expanded', ''));
+		toggleContainerDisplay(false);
+	}
+	expander.init(fixedEl, {
+		toggleSelector: `.${buttonClass}`
+	});
+	fixedEl.addEventListener('oExpander.expand', () => {
+		toggleContainerDisplay(true);
+		superstore.local.set(HAS_MINIMIZED, 0);
+		toggleSticky();
+	});
+	fixedEl.addEventListener('oExpander.collapse', () => {
+		toggleContainerDisplay(false);
+		superstore.local.set(HAS_MINIMIZED, 1);
+		toggleSticky();
+	});
+}
+
+// in addition to what o-expander gives us, add a class onto the container when changing expanded state
+function toggleContainerDisplay (showFull) {
+	if (showFull) {
+		fixedEl.setAttribute('class', fixedEl.className.replace(/\b\sn-welcome-banner--collapsed\b/g, ''));
+	} else {
+		fixedEl.setAttribute('class', fixedEl.className += ' n-welcome-banner--collapsed');
+	}
```

Comment:
Had a look at the polyfill, will use `classList` and its `add` and `remove` functions because no version of IE supports the second argument of toggle.